firefox-ios/Client/Frontend/Reader/View/ReaderModeStyleViewController.swift

```diff
@@ -6,26 +6,54 @@ import UIKit
 import Shared
 import Common
 
-// MARK: - ReaderModeStyleViewController
+class ReaderModeStyleViewController: UIViewController, Themeable, Notifiable {
+    public struct UX {
+        public static let stackViewSpacing: CGFloat = 8
+        public static let brightnessSize: CGFloat = 20
+        public static let brightnessMaxSize: CGFloat = 35
+        public static let sliderVerticalSpacing: CGFloat = 16
+        public static let brightnessHorizontalSpacing: CGFloat = 24
+        public static let separatorLineThickness: CGFloat = 1.0
+        public static let width: CGFloat = 270.0
+        public static let brightnessIconOffset: CGFloat = 10
+    }
 
-class ReaderModeStyleViewController: UIViewController, Themeable {
     // UI views
     private var fontTypeButtons: [ReaderModeFontTypeButton] = []
-    private lazy var fontSizeLabel: ReaderModeFontSizeLabel = .build()
     private var fontSizeButtons: [ReaderModeFontSizeButton] = []
     private var themeButtons: [ReaderModeThemeButton] = []
     private var brightnessImageViews = [UIImageView]()
     private var separatorLines = [UIView.build(), UIView.build(), UIView.build()]
 
-    private lazy var fontTypeRow: UIView = .build()
-    private lazy var fontSizeRow: UIView = .build()
+    private lazy var fontTypeStackView: UIStackView = .build { view in
+        view.axis = .horizontal
+        view.distribution = .fill
+        view.spacing = UX.stackViewSpacing
+        view.alignment = .center
+    }
+    private lazy var fontSizeStackView: UIStackView = .build { view in
```

Comment:
nit, we could have some spacing between the `lazy var` ? (the ones below that too)
```suggestion
    }

    private lazy var fontSizeStackView: UIStackView = .build { view in
```